test/cypress/data-grid/integration/formatting.cy.js

```diff
@@ -0,0 +1,85 @@
+describe('Remove the dashboard.', () => {
+    before(() => {
+        cy.visit('data-grid/v2/cell-formatting');
```

Comment:
It should not refer to v2 path. All demos should be in the datagrid.

Reply:
Yes, but the test updating is done in a different task.